packages/server/tests/acceptance/erc20.spec.ts

```diff
@@ -55,8 +55,8 @@ describe('ERC20 Acceptance Tests', async function () {
 
     before(async () => {
         accounts[0] = await servicesNode.createAliasAccount(30, relay.provider);
-        accounts[1] = await servicesNode.createAliasAccount(10, relay.provider);
-        accounts[2] = await servicesNode.createAliasAccount(10, relay.provider);
+        accounts[1] = await servicesNode.createAliasAccount(20, relay.provider);
```

Comment:
Q: Ouch, this makes tests more expensive.
Is there an option to share share logic accounts and reduce number of creates.
e.g. create account in index and use them in both `erc20` and `rpc` tests

Reply:
If we use shared accounts created in index.spec.ts they would still require roughly the same total amount of hbars, so I don't think this will be a worthwhile optimization.

Reply:
Okay